src/super_stream_consumer.ts

```diff
@@ -0,0 +1,53 @@
+import { Client } from "./client"
+import { Consumer, ConsumerFunc } from "./consumer"
+import { Offset } from "./requests/subscribe_request"
+
+export class SuperStreamConsumer {
+  private consumers: Map<string, Consumer> = new Map<string, Consumer>()
+  public consumerRef: string
+  private locator: Client
```

Comment:
is locator a good name for the client underneath the consumer?

Reply:
It is the locator, that's the term we are using for the client that manages the connections right?